src/midst_toolkit/evaluation/privacy/batched_eir.py

```
@@ -0,0 +1,168 @@
+
+import numpy as np
+import pandas as pd
+from scipy.stats import entropy
+from syntheval.metrics.core.metric import MetricClass
+from syntheval.utils.nn_distance import _knn_distance
+from tqdm.auto import tqdm
+
+
+def _column_entropy(labels: list | np.ndarray) -> np.number:
+    """Compute the entropy of a single column."""
+    value, counts = np.unique(np.round(labels), return_counts=True)
```

Ruff (0.14.10)

Unpacked variable `value` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)